__tests__/app.test.js

```diff
@@ -578,3 +578,36 @@ describe('Get /api/users', () => {
             });
     });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+    test("204: response with 204 status and no content", () => {
+        return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+            .then( res => {
+                expect(res.body).toEqual({});
+            });
```

Comment:
This assertion does not need to be made. Nothing can be send with a 204 status code, so this could never fail.